import-export-cli/integration/testutils/devFirst_testUtils.go

```diff
@@ -79,6 +80,7 @@ func ValidateInitializeProjectWithOASFlag(t *testing.T, args *InitTestArgs) {
 
 	t.Cleanup(func() {
 		base.RemoveDir(args.InitFlag)
+		base.WaitForIndexing()
```

Comment:
No need to wait here because we are only removing a directory and nothing related to indexing is taking place in APIM side.